flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/channel/ChannelStateWriter.java

```diff
@@ -73,10 +73,20 @@
 	void addOutputData(long checkpointId, ResultSubpartitionInfo info, int startSeqNum, Buffer... data);
 
 	/**
-	 * Finalize write of channel state for the given checkpoint id.
-	 * Must be called after {@link #start(long)} and all of the data of the given checkpoint added.
+	 * Finalize write of channel state data for the given checkpoint id.
+	 * Must be called after {@link #start(long)} and all of the input data of the given checkpoint added.
+	 * When both {@link #finishInput} and {@link #finishOutput} were called the results can be (eventually) obtained
+	 * using {@link #getWriteCompletionFuture}
 	 */
-	void finish(long checkpointId);
```

Comment:
we should also adjust the references of `{@link #finish(long)}` in above `addInputData` and `addOutput` descriptions for the commit "[FLINK-16744][task] split finish() in ChanStateWrite"